src/test/java/Homework17.java

```diff
@@ -0,0 +1,46 @@
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.safari.SafariDriver;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework17 extends BaseTest {
+
+    @Test
+    public void AddSongToPlaylist() throws InterruptedException {
+        openPage("https://bbb.testpro.io/");
+        //Log in
+        enterEmail("linulya1411@gmail.com");
+        enterPassword("te$t$tudent");
+        clickSubmit();
+        Thread.sleep(2000);
+        //Search for a song
+        searchForSong("Beautiful");
+        Thread.sleep(2000);
+        viewAllResults();
+        //Select the song
+        selectFirstSong();
+        //Add song to the playlist
+        clickAddTo();
+        choosePlaylist("My Playlist");
+
+        verifyNotificationMessage();
+
+
+        Assert.assertTrue(addedSongNotification());
+
+    }
+
+
+
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._